gatsby/howitworks.rst

```diff
@@ -25,7 +24,9 @@ It works with a basic rule: everything is a component, and every interface is bu
 Components let you split the UI into independent, reusable pieces, and think about each piece in isolation.
 Conceptually, components are like JavaScript functions. They accept arbitrary inputs (called “props”) and return React elements describing what should appear on the screen.
 
-In Gatsby all pages are React components that takes in input some data (if needed) from GraphQL and displays some informations.
+In Gatsby all pages are React components that takes in input some data (if needed) from GraphQL and displays some information.
```

Comment:
Refer to the documentation style guide and sub pages:
https://docs.plone.org/about/contributing/documentation_styleguide.html

s/takes/take and s/displays/display

Reply:
Throughout this document, each sentence should be one line without wrapping.

Reply:
@stevepiercy thanks for the tip. I wanted to do a full-review later, but you're right.

Can you explain your suggestion for _s/takes/take and s/displays/display_?

Reply:
sorry, it's late here..now i get it ;)

Reply:
😉